lnrpc/routerrpc/router_backend.go

```diff
@@ -382,6 +384,15 @@ func (r *RouterBackend) extractIntentFromSendRequest(
 	payIntent.PayAttemptTimeout = time.Second *
 		time.Duration(rpcPayReq.TimeoutSeconds)
 
+	// Route hints.
```

Comment:
Doesn't attempt to also parse the route hints from the regular payment request.

Reply:
Payment request route hints are appended to the list of hints further down. This allows you to augment the encoded hints with a few more.